### PR TITLE
add buttons above navbar

### PR DESCRIPTION
**Pull request by jpguisa**
Added docs, create account and signup links above navbar on the right corner

### CODE REVIEW

**Comment by akeller** [comment]
More context on the internal Camunda Slack - https://camunda.slack.com/archives/C6M6X3EKZ/p1684769980750839

FYI @pepopowitz

File: layouts/partials/header.html [hunk 1]
```diff
@@ -50,6 +50,12 @@
 <!-- End Google Tag Manager (noscript) -->
 
 <nav class="navbar navbar-fixed-top">
+  <div class="navbar-container navbar-top-section">
+    <span>Looking for Camunda Platform 8? >></span>
+    <a target="_blank" href="https://docs.camunda.io/">Docs</a>
+    <a target="_blank" href="https://signup.camunda.com/accounts?_ga=2.124018762.1966193156.1686668003-663653735.1686668003">Create account</a>
```
Comment (pepopowitz) [inline layouts/partials/header.html:56]: Based on https://github.com/camunda/camunda-platform-docs/pull/2287, I suspect we'd want to remove the `_ga=...` from this link, but it should be clarified with the Acquisition team.

File: styles/_navbar.less [hunk 2]
```diff
@@ -116,6 +117,37 @@
 body.site-menu-open .site-menu-toggle {
   .hamburger-button-open();
 }
+.navbar-top-section{
```
Comment (pepopowitz) [inline styles/_navbar.less:120]: I don't have specific code suggestions, but if we decide to stick with a second row of nav, there is still CSS work to be done here. Right now, the additional bar pushes the original bar down further, masking the top of the content. 

Here's what it looked like before these changes: 

image

And here's what these changes introduce: 

image

I'm unable to scroll far enough on the page to see those top bits.

Comment (jpguisa) [reply styles/_navbar.less:120]: This is fixed!

Comment (pepopowitz) [reply styles/_navbar.less:120]: 👍 

For others playing along, this is what it looks like now: 

image

**Review by pepopowitz — APPROVED** [review]

File: layouts/partials/header.html (hunk 1 above)
Comment (akeller) [reply layouts/partials/header.html:56]: @jpguisa can you adjust this link to `https://signup.camunda.com/accounts?utm_source=docs.camunda.org&utm_medium=referral` please?

**Comment by akeller** [comment]
@danielkelemen did you have more feedback on this? This is part of an Acquisition Team experiment. Sorry for not making that clearer earlier in the comments or description.

**Comment by akeller** [comment]
@danielkelemen any further feedback here?

**Comment by danielkelemen** [comment]
Hi @akeller,
The header is not visible on smaller screens.
No further feedback from me.

**Comment by akeller** [comment]
> Hi @akeller, The header is not visible on smaller screens. No further feedback from me.

@jpguisa can you please make adjustments here?

**Comment by jpguisa** [comment]
@akeller @danielkelemen On smaller screens the content won't fit correctly. How should I proceed?

image

**Comment by akeller** [comment]
@jpguisa My quick answer is just to **hide the nav you added on smaller screens.** This is the issue with having multiple navs when space is already limited. Since this is technically an experiment (or initially designed to be an experiment), hiding this nav for small screens is the best path forward. I will sync with the team internally on this as well.

**Comment by akeller** [comment]
@pepopowitz should the package-lock.json file be included in the PR?

**Comment by pepopowitz** [comment]
> @pepopowitz should the package-lock.json file be included in the PR?

No, it shouldn't, that's strange that it updated. @jpguisa can you please revert the changes to that file?

**Comment by jpguisa** [comment]
@pepopowitz Done!

**Review by akeller — APPROVED** [review]

**Comment by akeller** [comment]
Even though this is merged, the changes need to be added to the [static](https://github.com/camunda/camunda-docs-static) and [manual](https://github.com/camunda/camunda-docs-manual) repos. @jpguisa you can do this, or I'll add this to my list of this to do this week.

**Pull request merged**